components/Button/Button.test.jsx

```diff
@@ -0,0 +1,29 @@
+import jsdom from 'mocha-jsdom';
+import React from 'react';
+import { mount } from 'enzyme';
+import { expect } from 'chai';
+import Button from './button';
+
+describe('Button', () => {
+  jsdom();
+
+  it('Renders', () => {
+    const wrapper = mount(<Button>foo</Button>);
+    expect(wrapper.props().children).to.equal('foo');
+  });
+
+  it('Accepts click events', () => {
+    let callCount = 0;
+    const handleClick = () => callCount++;
+    const wrapper = mount(<Button onClick={handleClick}>foo</Button>);
+    expect(callCount).to.equal(0);
+    wrapper.find('button').simulate('click');
+    expect(callCount).to.equal(1);
+  });
+
+  it('Accepts arbitrary HTML attributes', () => {
+    const wrapper = mount(<Button height={20} id='123'>foo</Button>);
+    expect(wrapper.props().height).to.equal(20);
```

Comment:
I just realized that this test is useless (and broken), since we should be testing whether it applies arbitrary attributes to the html <button> element itself.

The test should be:

```js
expect(wrapper.find('button').node.getAttribute('height')).to.equal('20');
```

And it's broken because we currently don't allow arbitrary HTML attributes to be passed in directly as props. They have to be passed in through an `attrs` prop, which is less-than-ideal.

An easy fix would be to reuse the proposed `excludeProps` function in the `<Input>` PR.